clients/ctl/admin-ui/cypress/.eslintrc.json

```diff
@@ -1,7 +1,11 @@
 {
   "extends": ["plugin:cypress/recommended"],
+  "plugins": ["no-only-tests"],
```

Comment:
Being able to use `it.only` is helpful while developing with cypress mocked APIs, but you def don't want to commit them.